client/accept-invite/invite-message/index.jsx

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
I'd consider putting the `invite-message` component in the `client/signup` folder.